README.md

```
@@ -1,210 +1,216 @@
 
-**Deepak Nemade (DN)**
-- Professional System Administrator
-- DevOps Engineer
-- Security Specialist
-- Linux Expert
+**Deepak Nemade (DN)**  
+Professional System Administrator & DevOps Engineer
+
+## Repository
+
+**Star this repository**: https://github.com/DeepDN/Ubuntu_Scripts
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)